packages/mc-scripts/config/views/login.pug

```diff
@@ -0,0 +1,40 @@
+html
```

Comment:
Nice. Sorry to ask but do we really need pug for this? 😄

Reply:
We don't, but on the other hand...why not? 😄

We can use plain html, I don't mind...as long as it works